app/src/main/java/com/ciandt/dragonfly/example/features/projectselection/ProjectSelectionPresenter.kt

```diff
@@ -68,8 +68,8 @@ class ProjectSelectionPresenter(private val interactor: ProjectSelectionContract
         if (project.hasDownloadedVersion()) {
             val version = project.getLastDownloadedVersion()!!
 
-            val others = Bundle()
-            others.putBoolean("benchmark", project.showBenchmark)
+            val others = HashMap<String, Serializable>()
+            others.put("benchmark", project.showBenchmark)
```

Comment:
What is this for?

Reply:
This is a revert to the previous code.
Old code was using Map, but map is not serializable, so I changed to Bundle.
The problem is that Bundle breaks the unit test, because Bundle is from android package.
So, I researched and found that Hashmap is serializable, and now I reverted it.